components/dada-execute/src/step/apply_op.rs

```diff
@@ -41,12 +41,16 @@ impl Stepper<'_> {
         match (&self.machine[lhs], &self.machine[rhs]) {
             (&ObjectData::Bool(lhs), &ObjectData::Bool(rhs)) => match op {
                 Op::EqualEqual => Ok(self.machine.our_value(lhs == rhs)),
+                Op::GreaterEqual => Ok(self.machine.our_value(lhs >= rhs)),
+                Op::LessEqual => Ok(self.machine.our_value(lhs <= rhs)),
```

Comment:
Appling < and > to bools seems of questionable utility to me - that's why this match doesn't do it already. I didn't realize rust allowed these operators to be applied to bools. @nikomatsakis do you want < and > to apply to bools?

Reply:
I guess if it were to be used as keys in maps, then these operators would need to apply.

Reply:
I think it makes sense for the reason you said -- being usable in maps, or in sorted vectors, with binary search. It's often not bools on their own but rather some struct that contains a boolean field.